kconfig-lib.sh

```diff
@@ -1,20 +1,21 @@
+#!/bin/bash
 
 BUILD_DIR=$(pwd)
 
 # find merge_config in code directory
 FILE=scripts/kconfig/merge_config.sh
 if test -f "$FILE"; then
-    COMMAND=$FILE;
+    export COMMAND=$FILE;
```

Comment:
Again I believe this file is meant to be sourced so I don't think this `export` makes sense.

Reply:
if I don't use export shellcheck complains that the variable is not used.

````
In kconfig-lib.sh line 18:
            COMMAND=$SOURCE_DIR/$FILE;
            ^-----^ SC2034 (warning): COMMAND appears unused. Verify use (or export if used externally).
````

what the heck.

Reply:
Ideally shellcheck would analyze this file only _indirectly_ when sourced from another file - exactly like how it's being used.

In practice just ignore this warning.